### PR TITLE
Allow Notebook output to be preserved

### PR DESCRIPTION
**Pull request by FlorentinD**
main motivation: visualization notebook such as #766

### CODE REVIEW

File: examples/README.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Examples
+
+This folder contains example notebooks on how to use the `graphdatascience` python client.
+
+
+## Custom tooling for notebooks
```
Comment (adamnsch) [inline examples/README.adoc:6]: I wonder if this stuff should be under `Style guide` in `TESTING.md`?

Comment (adamnsch) [reply examples/README.adoc:6]: Which btw needs updating since changing to `ruff`

File: examples/README.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+As for the rest of this repository, you can run `../scripts/makestyle` and `../scripts/checkstyle` to apply and check the style of the notebook.
+By default, `makestyle` will remove all cell outputs. If you want to preserve some outputs, tag the cell with `preserve-output`.
```
Comment (adamnsch) [inline examples/README.adoc:9]: Nice! 🎉

File: examples/README.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+To generate documenatation, you can run `../scripts/nb2doc/convert`.
```
Comment (adamnsch) [inline examples/README.adoc:11]:
```suggestion
To generate documentation, you can run `../scripts/nb2doc/convert`.
```

**Review by adamnsch — APPROVED** [review]
Nice work!

File: examples/README.adoc (hunk 1 above)
Comment (FlorentinD) [reply examples/README.adoc]: done 🏓

**Pull request merged**